db/src/entities/stations.rs

```diff
@@ -79,9 +79,38 @@ pub struct StationChangeset {
 pub async fn load_all(
     executor: impl sqlx::Executor<'_, Database = Sqlite>,
 ) -> Result<Vec<Station>, crate::Error> {
-    let stations = sqlx::query_as!(Station, "SELECT id, name, uic, latitude, longitude, country, country_hint, info_de, info_en, info_es, info_fr, info_it, info_nb, info_nl, info_cs, info_da, info_hu, info_ja, info_ko, info_pl, info_pt, info_ru, info_sv, info_tr, info_zh FROM stations")
-        .fetch_all(executor)
-        .await?;
+    let stations = sqlx::query_as!(
+        Station,
+        "SELECT
+            id,
+            name,
+            uic,
+            latitude,
+            longitude,
+            country,
+            country_hint,
+            info_de,
+            info_en,
+            info_es,
+            info_fr,
+            info_it,
+            info_nb,
+            info_nl,
+            info_cs,
+            info_da,
+            info_hu,
+            info_ja,
+            info_ko,
+            info_pl,
+            info_pt,
+            info_ru,
+            info_sv,
+            info_tr,
+            info_zh
+        FROM stations"
```

Comment:
```suggestion
        FROM
            stations"
```

bit of a nitpick but I'd apply this pattern consistently (same for the others below)

Reply:
Np, I'll just add it on #92 if that's ok